packages/formstr-app/src/components/Routing/index.js

```diff
@@ -1,29 +1,34 @@
-import { Routes, Route } from "react-router-dom";
+import { Routes, Route, Navigate } from "react-router-dom";
 import App from "../V0/App";
 import NewForm from "../NewForm";
 import FillForm from "../FillForm";
 import ViewResponses from "../ViewResponses";
-import MyForms from "../MyForms";
 import { GlobalForms } from "../GlobalForms";
 import { DraftsController } from "../MyForms/DraftsController";
+import MyFormsV0 from "../MyForms";
 import CreateForm from "../../containers/CreateForm";
+import MyForms from "../../containers/MyForms";
+import PublicForms from "../../containers/PublicForms";
+import { ROUTES } from "../../constants/routes";
 
 function Routing() {
   return (
     <Routes>
       <Route path="/" element={<App />}>
         <Route path="global" element={<GlobalForms />} />
-        <Route path="myForms" element={<MyForms />} />
         <Route path="forms/new" element={<NewForm />} />
         <Route path="forms/fill" element={<FillForm />} />
         <Route path="forms/:npub" element={<FillForm />} />
         <Route path="forms/:nsec/responses" element={<ViewResponses />} />
         <Route path="drafts/:encodedForm" element={<DraftsController />} />
         <Route path="forms/responses" element={<ViewResponses />} />
         <Route path="*" element={<NewForm />} />
-        <Route index element={<MyForms />} />
+        <Route path="v0/myforms" element={<MyFormsV0 />} />
+        <Route index element={<Navigate to={ROUTES.MY_FORMS} />} />
       </Route>
-      <Route path="v1" element={<CreateForm />} />
+      <Route path={`${ROUTES.CREATE_FORMS}/*`} element={<CreateForm />} />
```

Comment:
should we have these as children to v1? just wondering

Reply:
yes, but anyway we are going to make v1 as default route in a few days, so instead of reworking now for v1, we can do the same for v0 once we complete our builder page.